util/topgen/intermodule.py

```diff
@@ -1003,9 +1003,19 @@ def im_netname(sig: OrderedDict, suffix: str = "", default_name=False) -> str:
             # custom default has been specified
             if obj["default"]:
                 return obj["default"]
-            if isinstance(sig["width"], Parameter):
-                return "{{{param}{{{package}::{struct}_DEFAULT}}}}".format(
-                    param=sig["width"].name_top,
+            repetitions = 1
+            if sig["width"]:
+                if isinstance(sig["width"], Parameter):
+                    repetitions = sig["width"].name_top
+                elif isinstance(sig["width"], str):
+                    repetitions = int(sig["width"])
+                elif isinstance(sig["width"], int):
+                    repetitions = sig["width"]
+                else:
+                    raise ValueError(f"Invalid inter-module signal width {sig['width']}")
```

Comment:
Nit: I'd probably put a colon after `width` to make the error message easier to read:
```python
                    raise ValueError(f"Invalid inter-module signal width: {sig['width']}")
```